src/test/java/org/cactoos/iterable/IterableOfTest.java

```diff
@@ -73,5 +81,69 @@ public void convertsObjectsToIterable() {
         );
     }
 
-}
+    // @todo #971:30min Implement equals() and hashCode() on IterableEnvelope.
+    //  Then, refactor this test so that only
+    //  `new IsEqual<>(new Joined<>(first, second, third))` is required as a
+    //  matcher.
+    @Test
+    @SuppressWarnings({"unchecked", "PMD.AvoidDuplicateLiterals"})
+    public void containAllPagedContentInOrder() throws Exception {
+        final Iterable<String> first = new IterableOf<>("one", "two");
+        final Iterable<String> second = new IterableOf<>("three", "four");
+        final Iterable<String> third = new IterableOf<>("five");
+        final Iterable<Iterable<String>> service = new IterableOf<>(
+            first, second, third
+        );
+        final Iterator<Iterable<String>> pages = service.iterator();
+        MatcherAssert.assertThat(
+            "must have all page values",
```

Comment:
@fabriciofx I hadn't noticed this. I will fix it. What is the reasoning behind this?

Reply:
@llorllale All sentences must start with a capital letter? Anyway, I'm just following the pattern, because all these sentences starts with a capital letter.

Reply:
@llorllale I checked *all* `assertThat` messages in cactoos and *almost* all start with a capital letter. Why *almost*? Only `MapEntryTest` and `MapEnvelopeTest` don't follow this pattern. I used this command to check: `grep -A1 "assertThat" * -r \| grep "\".*\"" \| grep "\"[a-z0-9].*\""`